awslambda/project.json

```diff
@@ -0,0 +1,39 @@
+{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "compilerName": "fsc",
+    "emitEntryPoint": false,
+    "compile": "*.fs",
+    "debugType": "portable",
+    "outputName": "<%= namespace %>"
```

Comment:
@slang25 glad you caught this, as the AWS Lambda handler name depends on assembly name it's worth being explicit about it, otherwise `dotnet build` uses the containing folder name